cmd/bytesizetostring_test.go

```diff
@@ -0,0 +1,103 @@
+package cmd
+
+import (
+	"testing"
+)
+
+func TestBToString(t *testing.T) {
```

Comment:
Thanks for the tests! Please consider using the `check` package for validation. Refer to other test files in the cmd module.